almdrlib/environment.py

```diff
@@ -99,16 +93,10 @@ def get(self, key, default=None, format='decoded', type=None):
     def get_parameter(self, key, default=None, decrypt=False):
         if "ssm" not in self.source:
             raise AlmdrlibSourceNotEnabledError("ssm is not enabled for this environment")
-        try:
-            parameter = self.ssm.get_parameter(Name=self._make_ssm_key(key), WithDecryption=decrypt)
-        except self.ssm.exceptions.ParameterNotFound:
-            return default
-        except botocore.exceptions.ClientError as e:
-            raise AlEnvException() from e
-        return parameter["Parameter"]["Value"]
+        return AlEnv._get_cached_ssm_parameter(self._make_ssm_key(key), default, decrypt)
 
     def _make_ssm_key(self, option_key):
-        return f"/deployments/{self.stack_name}/{self._get_region()}/env-settings/{self.application_name}/{self._make_client_option_key(option_key)}"
+        return f"/deployments/{self._get_stack_name()}/{self._get_region()}/env-settings/{self.application_name}/{self._make_client_option_key(option_key)}"
```

Comment:
Would you mind adding to https://github.com/alertlogic/alertlogic-sdk-python#readme the paths we search for in dynamodb / ssm.

Also, it's probably future work, but it would be nice to be able to specify those paths in `~/.alertlogic/config`